config/config-controllers.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-controllers
+  namespace: knative-eventing
+data:
+  # Expected to be a list of fully qualified resource names as keys and true or
+  # false. If empty, default to all known controllers.
+  # Example:
+  # subscription.eventing.knative.dev: true
```

Comment:
May be also add the list of known controllers here?  The user can then easily uncomment and configure the required ones. One downside is we need to keep this example updated each time.

Reply:
Why use a ConfigMap here if this is not reloadable? A simple string flag seems like it would be easier to manage.